### PR TITLE
Removes only_one_running.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Note that `lib/cdo/only_one.rb` is identical and in (widespread) usage.

THERE MAY BE TEST FAILURES (needing fixing) AS A RESULT OF TRANSITIVE DEPENDENCY ON PEGASUS ENV.

### CODE REVIEW

**Comment by breville** [comment]
Is this a reference that needs to be removed? https://github.com/code-dot-org/code-dot-org/blob/d344587164a1671898b827852c72e0e892918d79/lib/cdo/pegasus.rb#L10

File: lib/cdo/pegasus.rb [hunk 1]
```diff
@@ -7,7 +7,6 @@
 require_relative './pegasus/hash'
 require_relative './pegasus/object'
 require_relative './pegasus/string'
-require_relative './pegasus/only_one_running'
```
Comment (breville) [inline lib/cdo/pegasus.rb:10]: Oh, should this be replaced with a require of only_one?

Comment (ashercodeorg) [reply lib/cdo/pegasus.rb:10]: Ideally, not, as I don't consider `only_one` as being part of the pegasus environment. Thus my (current) comment in the PR summary.

Comment (ashercodeorg) [reply lib/cdo/pegasus.rb:10]: Unless you disagree, I would prefer to manually insert the dependency in all the places where it is used. Planning on following up with such another commit.

Comment (breville) [reply lib/cdo/pegasus.rb:10]: Seems fair to me.

**Comment by ashercodeorg** [comment]
PTAL.

**Comment by breville** [comment]
updates to explicitly cdo/only_one lgtm.

**Pull request merged**